app/main/routes.py

```diff
@@ -187,6 +189,33 @@ def record(record_id: uuid.UUID):
     return render_template("record.html", record=file_metadata)
 
 
+@bp.route("/download/<uuid:record_id>")
+@access_token_sign_in_required
+def download_record(record_id: uuid.UUID):
+    file = File.query.get_or_404(record_id)
+
+    validate_body_user_groups_or_404(
+        file.file_consignments.consignment_bodies.Name
+    )
+
+    s3 = boto3.client("s3")
+    bucket = current_app.config["RECORD_BUCKET_NAME"]
+    file_metadata = get_file_metadata(record_id)
+    consignment_reference = file_metadata["consignment"]
+    file_path = file_metadata["file_path"]
+    key = f'{consignment_reference}/{file_path.rstrip("/")}'
+    file_name = file_metadata["file_name"]
+    file = s3.get_object(Bucket=bucket, Key=key)
+
+    response = Response(
+        file["Body"].read(),
+        headers={"Content-Disposition": "attachment;filename=" + file_name},
+    )
+    if response.status_code == 200:
+        print("ok")  # success message will go here
```

Comment:
This response code of this will always be 200 since we are creating it above implicitly with the default status code which is 200.

I think the message should be shown on the record page, not on the download page. 

Will reach out to chat offline.